tests/test_loggers_line_size.py

```diff
@@ -66,50 +69,91 @@ def test_log_line_no_size_limit(self):
 
     @mock.patch('traceback.format_stack')
     @mock.patch('clog.loggers.ScribeLogger._log_line_no_size_limit')
     def test_warning_line_size(self, mock_log_line_no_size_limit, mock_traceback):
-        line = create_test_line(WARNING_LINE_SIZE_IN_BYTES)
-        assert len(line) > WARNING_LINE_SIZE_IN_BYTES
-        assert len(line) <= MAX_LINE_SIZE_IN_BYTES
+        line = create_test_line(WARNING_SCRIBE_LINE_SIZE_IN_BYTES)
+        assert len(line) > WARNING_SCRIBE_LINE_SIZE_IN_BYTES
+        assert len(line) <= MAX_SCRIBE_LINE_SIZE_IN_BYTES
         self.logger.log_line(self.stream, line)
         assert self.logger.report_status.called_with(
             False,
             'The log line size is larger than %r bytes (monitored in \'%s\')'
-            % (WARNING_LINE_SIZE_IN_BYTES, WHO_CLOG_LARGE_LINE_STREAM)
+            % (WARNING_SCRIBE_LINE_SIZE_IN_BYTES, WHO_CLOG_LARGE_LINE_STREAM)
         )
         assert mock_log_line_no_size_limit.call_count == 2
         call_1 = mock.call(self.stream, line)
         origin_info = {}
         origin_info['stream'] = self.stream
         origin_info['line_size'] = len(line)
+        origin_info['line_preview'] = line[1000:]
         origin_info['traceback'] = ''.join(mock_traceback)
         origin_info_line = json.dumps(origin_info).encode('UTF-8')
         call_2 = mock.call(WHO_CLOG_LARGE_LINE_STREAM, origin_info_line)
         mock_log_line_no_size_limit.assert_has_calls([call_1, call_2])
+
+
+@pytest.mark.acceptance_suite
+class TestCLogMonkLoggerLineSize(object):
+
+    @pytest.yield_fixture(autouse=True)
+    def setup(self):
+        self.stream = 'foo'
+        loggers.MonkProducer = mock.Mock()
+        self.logger = MonkLogger(
+            'clog_test_client_id',
+            stream_backend_map = {'foo': 'monk'}
+        )
+        self.logger.report_status = mock.Mock()
+
+    @mock.patch('clog.loggers.MonkLogger._log_line_no_size_limit')
+    def test_normal_line_size(self, mock_log_line_no_size_limit):
+        line = create_test_line()
+        self.logger.log_line(self.stream, line)
+        assert not self.logger.report_status.called
+        mock_log_line_no_size_limit.assert_called_once_with(self.stream, line)
+
+    @mock.patch('traceback.format_stack')
+    @mock.patch('clog.loggers.MonkLogger._log_line_no_size_limit')
+    def test_max_line_size(self, mock_log_line_no_size_limit, mock_traceback):
+        line = create_test_line(MAX_MONK_LINE_SIZE_IN_BYTES)
+        self.logger.log_line(self.stream, line)
+        assert self.logger.report_status.call_count == 1
+        assert mock_log_line_no_size_limit.call_count == 1
+        message_report = {
+            'stream': self.stream,
+            'line_size': len(line),
+            'line_preview': line[1000:],
+            'traceback': ''.join(mock_traceback),
+        }
+        message_report_line = json.dumps(message_report).encode('UTF-8')
+        mock_log_line_no_size_limit.called_with(
```

Comment:
s/`called_with`/`assert_called_with`/g

Too bad Python's magic mock allows you to call undefined methods like `called_with` 😞 To avoid this, you could add `autospec=True` to your `mock.patch` call. It's generally good practice to do so everywhere